### ISSUE TITLE
Book 05: Sentence 019

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#51]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/42d3cc86-f963-438a-af76-bf7146655626)

```
# sent_id = Book_05_-_The_Omens.txt:19
# text = In jquac yn, in aqujn tachcauh, in tachcauhchiuhtiuh, in tachcauh icatiuh vtlipan, vncan peoa, in qujmelaquaoa, in qujnchoqujztlapaloa, iteiccaoan, itlahoapavalvan: in vtli ipan qujmixtlamachtia, qujmjlvia.
# text[norm] = In ihcuac in, in aquin tachcauh, in tachcauhchiuhtiuh, in tachcauh icatiuh ohtlipan, oncan pehua, in quimelacuaohua, in quinchoquiztlahpalohua, iteiccahuan, itlahuapahualhuan: in ohtli ipan quimixtlamachtia, quimilhuia.
# text[orig] = Inj·quac¶yn·,·inaqujn·tachcauh·,·in·tach¶cauhchiuhtiuh·,·intachcauh·ica¶tiuh·vtlipan·,·vncan·peoa·,·in·quj¶melaquaoa·,·in·qujn·choqujztla¶paloa·,·iteiccaoan·,·itlahoapaval¶van·:·in·vtli·ipan·qujmixtla¶machtia·,·qujmjlvia·.
# tagged = 97.22%
# analysed = 83.33%
1	In	in	SCONJ	_	_	3	_	_	_
2	jquac	ihcuac	ADV	_	_	3	_	_	_
3	yn	in	DET|SCONJ	_	_	21	_	_	_
4	,	,	PUNCT	_	_	7	punct	_	_
5	in	in	DET|SCONJ	_	_	6	_	_	_
6	aqujn	aquin	PRON	_	_	7	_	_	_
7	tachcauh	tachcauh	NOUN	_	State=Abs	21	_	_	_
8	,	,	PUNCT	_	_	10	punct	_	_
9	in	in	SCONJ	_	_	10	_	_	_
10	tachcauhchiuhtiuh	_	VERB	_	_	7	_	_	_
11	,	,	PUNCT	_	_	14	punct	_	_
12	in	in	DET|SCONJ	_	_	14	_	_	_
13	tachcauh	tachcauh	NOUN	_	State=Abs	14	_	_	_
14	icatiuh	_	VERB	_	_	7	_	_	_
15	vtlipan	_	ADV	_	_	14	_	_	_
16	,	,	PUNCT	_	_	21	punct	_	_
17	vncan	_	ADV	_	_	18	_	_	_
18	peoa	pehua	VERB	_	_	_	_	_	_
19	,	,	PUNCT	_	_	21	punct	_	_
20	in	in	SCONJ	_	_	21	_	_	_
21	qujmelaquaoa	_	VERB	_	_	0	root	_	_
22	,	,	PUNCT	_	_	24	punct	_	_
23	in	in	SCONJ	_	_	24	_	_	_
24	qujnchoqujztlapaloa	_	VERB	_	_	21	_	_	_
25	,	,	PUNCT	_	_	26	punct	_	_
26	iteiccaoan	_	NOUN	_	_	24	_	_	_
27	,	,	PUNCT	_	_	28	punct	_	_
28	itlahoapavalvan	tlahuapahualli	NOUN	_	Number=Plur|Number[psor]=Sing|Person[psor]=3|State=Poss	26	_	_	_
29	:	:	PUNCT	_	_	33	punct	_	_
30	in	in	DET|SCONJ	_	_	33	_	_	_
31	vtli	ohtli	NOUN	_	State=Abs	32	nmod	_	_
32	ipan	_	NOUN	_	NounType=Relat	33	obl	_	_
33	qujmixtlamachtia	ixtlamachtia	VERB	_	_	21	_	_	_
34	,	,	PUNCT	_	_	35	punct	_	_
35	qujmjlvia	ilhuia	VERB	_	_	33	_	_	_
36	.	.	PUNCT	_	_	21	punct	_	_
```

**Comment by ftyers** [comment]
Update: 
![Uploading imatge.png…]()

**Comment by ftyers** [comment]
Questions:
* What is the relation between *pehua* and *quimelacuahua*? *pehua* is not transitive, so it can't be `ccomp`, it would be weird for it to be `aux`, because the complement is introduced by *in*, so perhaps `xcomp` where the subject has to be shared?
* What is the relation between *aquin* and *tachcauh*? I put `acl`, but there is no *in*, but there are subsequent *in*s
* Should any of those `parataxis` be `parataxis:shared` ?

**Comment by Lguyogiro** [comment]
> What is the relation between pehua and quimelacuahua? pehua is not transitive, so it can't be ccomp, it would be weird for it to be aux, because the complement is introduced by in, so perhaps xcomp where the subject has to be shared?

Yeah `xcomp`.  This is also how it is annotated in nhi/azz iirc.

> What is the relation between aquin and tachcauh? I put acl, but there is no in, but there are subsequent ins

I think `acl` is right.  molina has "in aquin" as "he/the person who...", and grepping all the books, "in aquin V" is really frequent whereas "in aquin in" is almost non-existent.

**Comment by ftyers** [comment]
We currently have:
- tachcauhchiuhtiuh (with *tachcauh* as an enhanced node)
- quinchoquiztlahpalohua (with *choquiz* as an enhanced node)

Does that sound right? 

Also, what are we calling `Directional` in the other languages? Should it be `Movement` or `Purposive`? 
```
10	tachcauhchiuhtiuh	chihua	VERB	_	Directional=And|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	7	conj	7:conj	_
```

**Comment by ftyers** [comment]
Ok, update:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/284bcf49-3ce7-4a7e-9559-00da5a905539)

**Comment by Lguyogiro** [comment]
> Also, what are we calling Directional in the other languages? Should it be Movement or Purposive?

The morph analyses for this in the other treebanks are not quite satisfactory imho and we need to update them for the next release probably. We currently use Movement=(And|Ven) for the directional prefixes (only "Ven" is currently tagged in the tbs). Is `Purposive` a feature/value in any other UD langs? The only thing there is we also need to accoutn for the directionality of the purposivity (e.g. -ti vs -qui).

**Comment by Lguyogiro** [comment]
> Ok, update: ![imatge](https://private-user-images.githubusercontent.com/449545/311318708-284bcf49-3ce7-4a7e-9559-00da5a905539.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMDkwMzgsIm5iZiI6MTcxMDEwODczOCwicGF0aCI6Ii80NDk1NDUvMzExMzE4NzA4LTI4NGJjZjQ5LTNjZTctNGE3ZS05NTU5LTAwZGE1YTkwNTUzOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMFQyMjEyMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NjQyNjZhNmY0ZDExYTBjNjA3Zjc5YWViNThhYzNkNTY4Mzc2OTMyOTJjM2U1YmQ5NzFmMjFhYmM3M2RkMDY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.97IJrxe5UeOVnbNUCUO5jk5onHbjl5NPyvZcCG81h-Y)

LGTM

**Comment by ftyers** [comment]
`Purposive` is not in any other UD, but we do need to come up with something, and standardise for the other treebs too. Merged in fec397f.
